tests/snapshots/src/main/generated/com/airbnb/epoxy/ActivityRecyclerPool.kt

```diff
@@ -39,8 +39,10 @@ import java.util.ArrayList
 
 internal class ActivityRecyclerPool {
 //             ^^^^^^^^^^^^^^^^^^^^ definition semanticdb maven . . com/airbnb/epoxy/ActivityRecyclerPool#
+//                                  display_name ActivityRecyclerPool
 //                                  documentation ```kt\ninternal final class ActivityRecyclerPool\n```
```

Comment:
In kt files the signature documentation remains in `documentation` for now because semanticdb-kotlinc does not provide the structured signature field and adds the signature to the SematicDB `documentation` field already.